src/js/components/CheckBox/stories/Temp.js

```diff
@@ -0,0 +1,179 @@
+import React, { useState } from 'react';
+
+import {
+  Box,
+  Grommet,
+  CheckBox,
+  FormField,
+  DataTable,
+  ThemeContext,
+} from 'grommet';
+import { grommet } from 'grommet/themes';
+import { deepMerge } from 'grommet/utils';
+import { hpe } from 'grommet-theme-hpe';
+
+// Source code for the data can be found here
+// https://github.com/grommet/grommet/blob/master/src/js/components/DataTable/stories/data.js
+import { columns, DATA } from '../../DataTable/stories/data';
+
+const controlledColumns = columns.map(col => ({ ...col }));
+delete controlledColumns[0].footer;
+delete controlledColumns[3].footer;
+delete controlledColumns[4].footer;
+delete controlledColumns[4].aggregate;
+
+const myTheme = deepMerge(hpe, {
+  checkBox: {
+    check: {
+      extend: props =>
+        props.inFormField ? `box-shadow: none` : `box-shadow: undefined`,
```

Comment:
Why do we still need this is doing the `focusIndicator` flag?

Reply:
Good call. The conditional can be removed as it is irrelevant and always false (it was a leftover from alternate approach). I will clean up.

The `box-shadow: undefined` is needed to undo the `box-shadow: none` which is in the HPE theme currently and would be removed.

Reply:
Done.